src/workloads/contrib/qe_range_testing/workloads/local/experiment2_c0_s1.yml

```diff
@@ -0,0 +1,266 @@
+            f_dec128_2: {^ConvertToDecimal: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/tenthousandths.txt", "sequential": true, "startFromLine": 75000}}}}
+            _id: {^Inc: {start: 0, multiplier: 12500}}
+    - {Nop: true}
+
+  - Name: InsertActor_Thread7
+    Type: CrudActor
+    Threads: 1
+    Database: *encrypted_db
+    ClientName: *encrypted_pool
+    Phases:
+    - Repeat: 12500
+      Collection: *encrypted_coll
+      MetricsName: "load"
+      Operations:
+      - OperationName: insertOne
+        OperationMetricsName: inserts
+        OperationCommand:
+          Document:
+            f_sint32_1: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/ones.txt", "sequential": true, "startFromLine": 87500}}}}
+            f_sint32_2: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/ones.txt", "sequential": true, "startFromLine": 87500}}}}
+            f_bin64_1: {^ConvertToDouble: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/ones.txt", "sequential": true, "startFromLine": 87500}}}}
+            f_bin64_2: {^ConvertToDouble: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/tenthousandths.txt", "sequential": true, "startFromLine": 87500}}}}
+            f_dec128_1: {^ConvertToDecimal: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/ones.txt", "sequential": true, "startFromLine": 87500}}}}
+            f_dec128_2: {^ConvertToDecimal: {from: {^ChooseFromDataset:{"path": "./src/workloads/contrib/qe_range_testing/misc/tenthousandths.txt", "sequential": true, "startFromLine": 87500}}}}
+            _id: {^Inc: {start: 0, multiplier: 12500}}
+    - {Nop: true}
+
+
+  # We have a separate single-thread actor, each of which performs a different 1/(query_threads) of the queries.
+  
+  - Name: PreDefinedRangeQueryActor_Thread0
+    Type: CrudActor
+    Threads: 1
+    Database: *encrypted_db
+    ClientName: *encrypted_pool
+    Phases:
+    - *nop
+    - Repeat: 10000
+      Collection: *encrypted_coll
+      MetricsName: "query"
+      Operations:
+      - OperationName: find
+        OperationMetricsName: range_query
+        OperationCommand:
+          Filter: { : {
+              $gte: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "", "sequential": true, "startFromLine": 0}}}},
+              $lte: {^ConvertToInt32: {from: {^ChooseFromDataset:{"path": "", "sequential": true, "startFromLine": 0}}}}
+            }
+          }
+
+ 
+
+
+  - Name: LoggingActor0
+    Type: LoggingActor
+    Threads: 1
+    Phases:
+      - {Nop: true}
+      - Phase: 1
+        LogEvery: 5 minutes
+        Blocking: None
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - single-replica-fle
+    branch_name:
+      $gte: v7.0
```

Comment:
newline at end of file please.

Reply:
these are autogenerated, so they will never be committed.